lib/plugins/package/lib/packageService.js

```diff
@@ -26,9 +26,13 @@ module.exports = {
 
   getExcludes(exclude) {
     const packageExcludes = this.serverless.service.package.exclude || [];
-
+    // add local service plugins Path
+    const pluginsLocalPath = this.serverless.service.plugins &&
+      this.serverless.service.plugins.localPath;
+    const localPathExcludes = pluginsLocalPath &&
+      typeof pluginsLocalPath === 'string' ? [pluginsLocalPath] : [];
```

Comment:
You should use lodash's `_.isString()` here

Reply:
Done!